hieunc-guestbook-dojo/static/js/guestbook/models/Greeting.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by hieunc on 25/11/2015.
+ */
+
+define([
+        "dojo/_base/declare",
+        "dojo/_base/lang"
+    ], function (declare, lang) {
+        return declare(null, {
+            greeting_id: -1,
```

Comment:
- sai convention, hãy đặt tên `greetingId`
- check lại toàn bộ js/html/css: đặt tên biến và method theo kiểu `camelCase`: `dayLaTenBien`